modules/admission-controller-about.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-controller-about_{context}"]
+= About admission controllers
+
+Admission controllers are used to help regulate how {product-title} {product-version} functions.  Admission control plug-ins intercept requests to the master API to validate resource requests and ensure policies are adhered to. For example, admission control plug-ins are commonly used to enforce security policy, resource limitations or configuration requirements.
+
+Admission control plug-ins run in an admission chain, starting with mutating controllers that can modify resources, followed by validating controllers. Some controllers can both modify and validate. If any controller in the sequence rejects a request, the whole chain is aborted and an error is returned. Mutating controllers run prior to validating controllers so that modifications that they trigger can also be validated.
+
+{product-title} has a default set of plug-ins enabled for each resource type. These are required for proper functioning of master nodes. In addition, administrators can define custom admission webhooks to call webhook servers that enforce custom policies outside of these defaults.
+
+Some mutating admission controllers can produce side effects on resources related to the target object. In such situations, you must take steps to validate that the end result is as expected.
+
+[WARNING]
+====
+Custom admission control webhooks should be implemented cautiously because they impact cluster control plane operations. When implementing admission control webhooks in {product-title} {product-version}, ensure that you have read the documentation fully, tested for side effects of mutating webhooks, and implemented reclamation and reconciliation steps where appropriate.
```

Comment:
reclamation of what?

Reply:
I have rephrased this to be clearer. This was intended to state that steps must be included to restore resources back to their original state prior to mutation, in the event that a request does not pass through the entire admission chain.